modules/oadp-monitoring.adoc

```diff
@@ -0,0 +1,378 @@
+|Total number of attempted backups
+|Counter
+
+|velero_backup_deletion_attempt_total
+|Total number of attempted backup deletions
+|Counter
+
+|velero_backup_deletion_failure_total
+|Total number of failed backup deletions
+|Counter
+
+|velero_backup_deletion_success_total
+|Total number of successful backup deletions
+|Counter
+
+|velero_backup_duration_seconds
+|Time taken to complete backup, in seconds
+|Histogram
+
+|velero_backup_failure_total
+|Total number of failed backups
+|Counter
+
+|velero_backup_items_errors
+|Total number of errors encountered during backup
+|Gauge
+
+|velero_backup_items_total
+|Total number of items backed up
+|Gauge
+
+|velero_backup_last_status
+|Last status of the backup. A value of 1 is success, 0
+|Gauge
+
+|velero_backup_last_successful_timestamp
+|Last time a backup ran successfully, Unix timestamp in seconds
+|Gauge
+
+|velero_backup_partial_failure_total
+|Total number of partially failed backups
+|Counter
+
+|velero_backup_success_total
+|Total number of successful backups
+|Counter
+
+|velero_backup_tarball_size_bytes
+|Size, in bytes, of a backup
+|Gauge
+
+|velero_backup_total
+|Current number of existent backups
+|Gauge
+
+|velero_backup_validation_failure_total
+|Total number of validation failed backups
+|Counter
+
+|velero_backup_warning_total
+|Total number of warned backups
+|Counter
+
+|velero_csi_snapshot_attempt_total
+|Total number of CSI attempted volume snapshots
+|Counter
+
+|velero_csi_snapshot_failure_total
+|Total number of CSI failed volume snapshots
+|Counter
+
+|velero_csi_snapshot_success_total
+|Total number of CSI successful volume snapshots
+|Counter
+
+|velero_restore_attempt_total
+|Total number of attempted restores
+|Counter
+
+|velero_restore_failed_total
+|Total number of failed restores
+|Counter
+
+|velero_restore_partial_failure_total
+|Total number of partially failed restores
+|Counter
+
+|velero_restore_success_total
+|Total number of successful restores
+|Counter
+
+|velero_restore_total
+|Current number of existent restores
+|Gauge
+
+|velero_restore_validation_failed_total
+|Total number of failed restores failing validations
+|Counter
+
+|velero_volume_snapshot_attempt_total
+|Total number of attempted volume snapshots
+|Counter
+
+|velero_volume_snapshot_failure_total
+|Total number of failed volume snapshots
+|Counter
+
+|velero_volume_snapshot_success_total
+|Total number of successful volume snapshots
+|Counter
+
+|===
+
+[id="Viewing-metrics-OpenShift-Observe-UI_{context}"]
+==  Viewing metrics using the OpenShift Observe UI
+
+The metrics can be viewed in the OpenShift Administrator or Developer view, which must have access to the `openshift-adp project`.
+
+.Procedure
+
+. Select *Observe* and then select *Metrics*.
+.. In the *Developer* view select *Custom query*, or click on the *Show PromQL* link, type the query and click *Enter*.
+.. In the *Administrator* view type the Expression in the text field and select *Run Queries*.
```

Comment:
Technically since this is a one-step procedure (even though it has substeps), it should be an asterisk instead of `.` for the first step. There seemed to be a lot of steps jammed together though past that, so I took a crack at sorting it out, as a suggestion, though it does get a little wonky with the back-and-forth unordered/ordereds:

```suggestion
* Navigate to the *Observe* -> *Metrics* page:

** If you are using the *Developer* perspective, follow these steps:
.. Select *Custom query*, or click on the *Show PromQL* link.
.. Type the query and click *Enter*.

** If you are using the *Administrator* perspective, type the expression in the text field and select *Run Queries*.
```

Side question: is it really "clicking" `Enter`, or does it mean pressing the Enter key? I didn't see an **Enter** button in the screenshot, so I wondered.